tests/testthat/test-qenv_get_code.R

```diff
@@ -649,26 +644,29 @@ testthat::test_that("understands @ usage and do not treat rhs of @ as objects (o
     "a@x <- x@a"
   )
   q <- qenv()
-  q@code <- code # we don't use eval_code so the code is not run
+  code_split <- as.list(split_code(paste(code, collapse = "\n")))
```

Comment:
we don't test private utils. If it doesn't work with `eval_code` then we should test for error

Reply:
I am testing the code_parser/graph_parser part in here as I think it's pretty important to test how it understands `@`. 
If you don't eval the code, and only analyze the code as a character this test make sense. For `teal_data` it was possible to create `teal_data` with the code but without the code execution. For `qenv` this is not possible. I would like to see how parser handles `@` in tests

Reply:
If it is the case for `teal_data` then the test shouldn't be here ;]

Reply:
Please ignore my comment then. It is fine, I'll highlight this another time when somebody would touch this part ;]

Reply:
coolio